main.tex

```diff
@@ -126,6 +126,7 @@ \section{Introduction} \label{sec:intro}
 The \astropy project aims to provide an open-source and open-development core
 package (\astropypkg) and an ecosystem of \emph{affiliated packages} that
 support astronomical functionality in the \python programming language.
+``Open-development'' describes a process where anybody with an internet connection can suggest changes to the source code and contribute his or her opinion when new features, bug fixes, or other code changes are discussed (see Sect.~\ref{sect:devmod} and \ref{sect:ape} of how this is organized in practice).
```

Comment:
* Maybe also emphasize that anybody can __and__ welcome to suggest changes and make contributions etc.

* I usually prefer "they" instead of "his or her", but I am OK with the current version, too if more people prefer this way.